mp2-v1/tests/common/cases/table_source.rs

```diff
@@ -1496,13 +1512,43 @@ impl OffChainTableArgs {
         table_name: String,
         secondary_index_column: ColumnMetadata,
         non_indexed_columns: Vec<ColumnMetadata>,
+        provable_data_commitment: bool,
     ) -> Self {
         Self {
             table_name,
             secondary_index_column,
             non_indexed_columns,
             row_values: vec![], // instantiate with no rows
             last_update_epoch: 0,
+            provable_data_commitment,
+        }
+    }
+
+    pub(crate) async fn expected_root_of_trust(&self, table: &Table) -> Result<HashOutput> {
```

Comment:
why is this only pub(crate) ? that sounds like a nice API for the user to have?

Reply:
It's in the integration test crate, so I think it doesn't matter since no APIs are exported